lib/database/migrations/20240119144006-create-simulation-passes-runs.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
```

Comment:
Same, no need to create separate migration files

Reply:
merged